src/audio/host.c

```diff
@@ -346,6 +346,18 @@ static uint32_t host_get_copy_bytes_normal(struct comp_dev *dev)
 	 */
 	copy_bytes = ALIGN_DOWN(copy_bytes, hd->dma_copy_align);
 
+	if (!copy_bytes) {
```

Comment:
@ktrzcinx this is a very useful change!

Reply:
Agreed, this should reduce my number of extra debug prints!